pages/common/openttd.md

```diff
@@ -0,0 +1,28 @@
+# openttd
+
+> Open source clone of the Microprose game "Transport Tycoon Deluxe".
+> More information: <https://www.openttd.org>.
+
+- Load new/save game at start:
+
+`openttd -g {{path/to/file}}`
+
+- Start with a set resolution (eg. 1920x1200):
+
+`openttd -r {{resolution}}`
+
+- Start with a custom configuration file:
+
+`openttd -c {{path/to/file}}`
+
+- Start with selected drivers:
```

Comment:
>[G](https://github.com/tldr-pages/tldr/pull/5765/files#r614221046)iven that this command doesn't support long options, let's not do this, actually, because the placeholder words clarify which option controls which drivers.

...but maybe we can add a little bit more clarification:
```suggestion
- Start with selected video, sound, and music drivers:
```